Sources/Mocker/Mock.swift

```diff
@@ -34,8 +34,14 @@ public struct Mock: Equatable {
 
     public typealias OnRequest = (_ request: URLRequest, _ httpBodyArguments: [String: Any]?) -> Void
 
-    /// The type of the data which is returned.
-    public let dataType: DataType
+    /// The type of the data which designates the Content-Type header.
+    @available(*, deprecated, message: "Calling this property is unsafe. Use `contentType` instead.")
```

Comment:
It's only unsafe after calling it when moving to the new `contentType` initializers, right? Perhaps it would be neat to make that more explicit.
```suggestion
    @available(*, deprecated, message: "Calling this property is unsafe after migrating to the `contentType` initializers, and will be removed in an upcoming release. Use `contentType` instead.")
```